automation/test.sh

```diff
@@ -372,27 +372,33 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]|\\[storage-req\\]"
     export KUBEVIRT_E2E_SKIP="Migration"
   elif [[ $TARGET =~ vgpu.* ]]; then
-    export KUBEVIRT_E2E_FOCUS=MediatedDevices
+    export KUBEVIRT_E2E_FOCUS="MediatedDevices"
   elif [[ $TARGET =~ sig-compute-realtime ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-compute-realtime\\]"
   elif [[ $TARGET =~ sig-compute-migrations ]]; then
     export KUBEVIRT_E2E_FOCUS="Migration"
-    export KUBEVIRT_E2E_SKIP="GPU|MediatedDevices"
+    export KUBEVIRT_E2E_SKIP="MediatedDevices"
+    export ginko_params="$ginko_params --label-filter=\"!needs-gpu\""
   elif [[ $TARGET =~ sig-compute ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-compute\\]"
-    export KUBEVIRT_E2E_SKIP="GPU|MediatedDevices|Migration"
+    export KUBEVIRT_E2E_SKIP="MediatedDevices|Migration"
+    export ginko_params="$ginko_params --label-filter=\"!needs-gpu\""
   elif [[ $TARGET =~ sig-monitoring ]]; then
       export KUBEVIRT_E2E_FOCUS="\\[sig-monitoring\\]"
   elif [[ $TARGET =~ sig-operator ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-operator\\]"
   elif [[ $TARGET =~ sriov.* ]]; then
-    export KUBEVIRT_E2E_FOCUS=SRIOV
+    export KUBEVIRT_E2E_FOCUS="SRIOV"
```

Comment:
Adding the braces in this commit makes it harder to review only the parts that actually are relevant to dropping the GPU focus. Could you please extract such changes to a different commit?

Reply:
@EdDev By extracting the changes, you mean should I modify that commit? - like in here (https://stackoverflow.com/questions/1186535/how-to-modify-a-specified-commit)